templates/vue/src/components/modals/NodeModal/forms/ContentForm/MultipleChoiceForm.vue

```diff
@@ -0,0 +1,274 @@
+<template>
+  <div>
+    <b-form-group
+      :label="
+        multipleAnswerSelected ? 'Multiple Answers Form' : 'Single Answer Form'
+      "
+    >
+      <b-form-group>
+        <b-form-checkbox v-model="useImages" data-qa="multiplechoice-thumbnail">
```

Comment:
Multiple choice is two words so this should be
```suggestion
        <b-form-checkbox v-model="useImages" data-qa="multiple-choice-thumbnail">
```